flex/http.py

```diff
@@ -15,6 +16,12 @@
 except ImportError:
     _tornado_available = False
 
+_falcon_available = True
```

Comment:
Thanks for this.  I didn't realize the above tornado stuff was done the same way.  Appreciate you cleaning it up as well.